src/Examples.ts

```diff
@@ -151,6 +151,7 @@ export async function resolveTemplateArgAsync(
       // @ts-ignore
       repoUrl = new URL(template);
     } catch (error) {
+      // @ts-ignore
```

Comment:
```suggestion
      // @ts-expect-error
```

I [would prefer a `ts-expect-error`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#-ts-expect-error-comments), so that it pops up again if it's not needed anymore.